2022.9.0 (#24029)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

The output of this selector is the number of mired selected, for example, `243`.

## Config entry selector

> The config entry selector allows to user to select a configured integration

Reviewer: ```suggestion
The config entry selector allows the user to select an integration
```
